src/main/java/Duke.java

```diff
@@ -1,10 +1,133 @@
+import DukeException.Command_Not_Exist;
+import DukeException.No_Description;
+import Task.Task;
+
+import java.util.Scanner;
+import DukeException.*;
+import Task.*;
 public class Duke {
+    public static Boolean test_existence(Task[] tasks_list, String instruction){
+        for(Task element:tasks_list){
+            if(element==null)
+                break;
+            if(element.get_description().equals(instruction))
+                return true;
+        }
+        return false;
+    }
     public static void main(String[] args) {
+        //init the parameters to be used
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
+        String name ="Altria";
+        String prefix = "\n\t";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello, I'm "+name);
+        System.out.println("What can I do for you?");
+        Scanner in = new Scanner(System.in);
+        String instruction = " ";
+
+        int number_of_task = 0;
+        Task[] tasks_list = new Task[100];
+
+        while(true) //main loop of the chat bot
+        {
+            instruction = in.nextLine();
+            String[] split_instruction = instruction.split(" ", 2);
+
+            if(instruction.equals("bye")) //end condition
```

Comment:
```suggestion
            if (instruction.equals("bye")) //end condition
```

Should include space for readability purposes